app/circleSettings.js

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from 'react';
+import { SafeAreaView, Text, View, TouchableOpacity, Animated, ScrollView } from 'react-native';
+import Modal from "react-native-modal";
+import { styled } from 'nativewind';
+import { Button } from '../components/Buttons';
+import { Member } from '../components/Member.js';
+
+const StyledSafeArea = styled(SafeAreaView);
+const StyledView = styled(View);
+const StyledText = styled(Text);
+const StyledAnimatedView = styled(Animated.createAnimatedComponent(View));
+const StyledScrollView = styled(ScrollView);
+
+export default function Page() {
+
+	const [modalVisible, setModalVisible] = useState(false);
+
+	const [isEnabled, setIsEnabled] = useState(false);
+	const toggleSwitch = () => setIsEnabled(previousState => !previousState);
+	const togglePosition = React.useRef(new Animated.Value(1)).current;
+
+	const Trevor = "https://media.licdn.com/dms/image/C4E03AQEjKbD7qFuQJQ/profile-displayphoto-shrink_200_200/0/1574282480254?e=1701907200&v=beta&t=1BizKLULm5emiKX3xlsRq7twzFTqynOsfTlbRwqNuXI";
+
+	React.useEffect(() => {
+		Animated.timing(togglePosition, {
+		toValue: isEnabled ? 45 : 5,
+		duration: 200,
+		useNativeDriver: false,
+		}).start();
+	}, [isEnabled]);
+
+	return (
+		<StyledSafeArea className='bg-offblack border' style={{ flex: 1 }}>
+			<StyledView className='flex-1 items-center'>
+				<Button
```

Comment:
Like I mentioned above, the `sticky` class is not really doing anything here. Also the top bar is again misplaced on my devices which I think traces back to the layout/safearea rendering. something we need to look into